gctf/protocols/protocol_gctf.py

```diff
@@ -300,7 +348,13 @@ def _restimateCTF(self, ctfId):
         mic = ctfModel.getMicrograph()
         micFn = mic.getFileName()
         micDir = self._getMicrographDir(mic)
-        micFnCtf = self._getTmpPath(pwutils.replaceBaseExt(micFn, 'ctf'))
+
+        if gctf.Plugin.getActiveVersion() not in ['0.50', '1.06']:
```

Comment:
This code seems duplicated. ;)

Reply:
ehm, yes, it is similar to estimateCTF func